content/guides/monitors.md

```diff
@@ -49,15 +49,47 @@ on the left. This guide will walk through the configuration of the Metric type.
 
 ### Choose what to monitor
 
-1. Select the metric and scope you want to monitor.
+1. Select the alert type.
+    {{< img src="monitor/alert_type.png" alt="alert type" >}}
+
+    A **threshold alert** will compare the value in the selected
+    timeframe against a given threshold. There are additional options available
+    in the alerting conditions section. This is the standard alert case
+    where you know what sort values are unexpected.
+
+    A **change alert** will compare the change or % change of a value between
```

Comment:
"A **change alert** compares the absolute or percentage change in value between now and some time ago "